doc: migration-guide-3.7: Add IPv4 netmask information (#69763)
zephyrproject-rtos/zephyr: doc/releases/migration-guide-3.7.rst

  the long-duration zperf test show with correct throughput result.
  (:github:`69500`)

> * The IPv4 netmask is tied to one specific IPv4 address in the
>   network interface instead of being global to the whole interface.
>   If there is only one IPv4 address specified to the network interface,
>   nothing changes from user point of view. But if there are more than
>   one IPv4 address / network interface, the netmask must be specified
>   to each IPv4 address separately. (:github:`68419`)

Reviewer: The description confused me a little, maybe:

```suggestion
* Each IPv4 address assigned to a network interface has an IPv4 netmask
  tied to it instead of being set for the whole interface.
  If there is only one IPv4 address specified for a network interface,
  nothing changes from the user point of view. But, if there is more than
  one IPv4 address / network interface, the netmask must be specified
  for each IPv4 address separately. (:github:`68419`)
```
